### ISSUE TITLE
Following the GunProxy Ubuntu Service Guide

### ISSUE THREAD

**Issue by kinnth** [taniman/gunbotproxycommunity#18]
unfortunately I found the Ubunutu service guide hard to follow at the end when issuing all of the commands.  In some way something is not correct as the service is now possibly running but it's not working and i'm now not sure how to stop it / check.

Could you update the guide to

1. List each command on a separate line
2. List how to start / stop the service once created
3. Do a double check / sanity check line on the service before starting the service first time to make sure it will work once started.

Having an issue where the proxy keeps stopping if my terminal breaks.

**Comment by jayp0** [comment]
1. it seems fixed
2. systemctl start gbproxy, systemctl stop gbproxy  (it automatically starts at boot) 
3. All you need to check/change is these two in gbproxy.service and its kind of self explanatory.  
change /PATH/TO to match where proxy is located and rename(or symlink) GunbotProxyCommunity-0.9.x.jar to GunbotProxyCommunity.jar 
```
WorkingDirectory=/PATH/TO/GunbotProxy
ExecStart=/usr/bin/java -jar GunbotProxyCommunity.jar
```
If you already installed gbproxy.service and changed it afterward, you need to run following commands,
systemctl daemon-reload
systemctl restart gbproxy

btw, if you are running older version(14.04), this won't work since there's no systemd.  
Also, make sure you have installed Java.  (apt install default-jre-headless on 16.04 and up)
